app/src/main/res/drawable/bg_black_gray_border.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shape xmlns:android="http://schemas.android.com/apk/res/android"
-    android:shape="rectangle">
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
If we use `<ripple>` instead of `<layer-list>` this appears to produce the desired effect for all the buttons and as an added bonus the tests no longer crash. For example:

```xml
<?xml version="1.0" encoding="utf-8"?>
<ripple xmlns:android="http://schemas.android.com/apk/res/android"
    android:color="?android:colorControlHighlight">
    <item>
        <shape xmlns:android="http://schemas.android.com/apk/res/android"
            android:shape="rectangle">

            <solid
                android:color="@color/color_primary" />

            <stroke
                android:width="1dp"
                android:color="@color/gray" />
        </shape>
    </item>
</ripple>
```